package.json

```
@@ -58,7 +64,7 @@
       "ts"
     ],
     "rootDir": "src",
-    "testRegex": ".*\\.spec\\.ts$",
+    "testRegex": ".*\\.spec\\.ts$|.*\\.e2e-spec\\.ts$",
```

Since this is regex, you can use this instead
```suggestion
    "testRegex": ".*\\.(spec|e2e-spec)\\.ts$",
```